### PR TITLE
Feature/voteboat

### PR DESCRIPTION
**Pull request by RobvandenBerg**
Implementing the new voatboat stuff (flags instead of rolling)

### CODE REVIEW

File: app/assets/scripts/main.js [hunk 1]
```diff
@@ -307,32 +289,15 @@ function stepgame(timestamp) {
     }
   }
   
-  if(animalRollTime != 0)
+  if(animalVoteTime && animalVoteTime > timestamp - 5000)
```
Comment (mitchellolsthoorn) [inline app/assets/scripts/main.js:292]: Use constant

File: app/assets/scripts/main.js [hunk 2]
```diff
@@ -263,20 +246,19 @@ function stepgame(timestamp) {
...
+		vote('RIGHT', timestamp);
 		flickingDisabled = timestamp + 500;
 	  }
-	  if(isFlickingLeft() && animalJump == 0 && animalFall == 0 && animalRollTime == 0 || doRoll)
+	  if(isFlickingLeft() && animalJump == 0 && animalFall == 0)
 	  {
-		doRoll = false;
-		roll('LEFT', timestamp);
+		vote('LEFT', timestamp);
 		flickingDisabled = timestamp + 500;
```
Comment (mitchellolsthoorn) [inline app/assets/scripts/main.js:262]: Use constant

File: app/assets/scripts/resourceloading.js [hunk 3]
```diff
@@ -187,6 +191,8 @@ function loadResources()
 	setLoadableImage("/assets/images/up_arrow.png",1);
```
Comment (mitchellolsthoorn) [inline app/assets/scripts/resourceloading.js]: Use spaces after comma

**Comment by mitchellolsthoorn** [comment]
Use more comments and fix the stylchecks

**Comment by RobvandenBerg** [comment]
Will do checkstyle in a seperate pull request

**Pull request merged**